SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -886,11 +899,6 @@ void USpatialSender::SendCreateEntityRequest(USpatialActorChannel* Channel)
 	Receiver->AddPendingActorRequest(RequestId, Channel);
 }
 
-void USpatialSender::SendDeleteEntityRequest(Worker_EntityId EntityId)
```

Comment:
This method has been renamed RequestEntityDeletion to indicate via the API that we are no longer guaranteed a deletion will take place. The new method, RequestEntityDeletion, will switch internally based on whether or not the deletion is for a startup actor. In the case it is, it will flag the tombstone component. Otherwise it will proceed as normal with a SendDeleteEntityRequest.

Reply:
RequestEntityDeletion additionally has a boolean flag allowing us to force entity deletion. This can be used specifically for cases where the NetDriver (for example) is being torn down and we want to force deletion of a specific entity.

Reply:
Right now some other code like the SnapshotManager directly calls SendDeleteEntityRequest() on the underlying connection.  At some point we should probably review the APIs and route these types of calls through a common path.